### PR TITLE
moving Elliot and Robert to alumni section 

### PR DESCRIPTION
**Pull request by Baaaaam**

### CODE REVIEW

File: source/people/main.rst [hunk 1]
```diff
@@ -106,7 +97,9 @@ None at this time.
 Alumni
 -------------
 * `Tae Wook Ahn <http://www.oaep.go.th/index_en.php>`_, MS(2010), Exelon Nuclear
+* Elliott Biondo, PhD(2016), Idaho National Laboratory
```
Comment (katyhuff) [inline source/people/main.rst]: Are these switched? I thought Elliott was going to ORNL and Robert was going to INL

**Comment by Baaaaam** [comment]
oups, I fix it....

**Comment by katyhuff** [comment]
:+1:

**Comment by gonuke** [comment]
Looking good. Thanks @Baaaaam

**Pull request merged**